### ISSUE TITLE
Is it possible to do bulk signing and entry creation in the transparency log

### ISSUE THREAD

**Issue by arlandgoh** [sigstore/rekor#2304]
Would it be possible to sign the multiple artifact in single api call e.g. /api/v1/log/entries?

**Comment by Hayden-IO** [comment]
No, there's no API support for that. It's hard to offer SLOs for batch APIs as well so it's unlikely one would be added.
